common/djangoapps/philu_commands/management/commands/tests/test_mailchimp_commands.py

```diff
@@ -58,7 +58,8 @@ def test_sync_users_with_mailchimp_command(self, mocked_connection, mock_func):
         users = list(User.objects.all())
         users_data = Command().get_users_data_to_send(users)
         call_command('sync_users_with_mailchimp')
-        mock_func.assert_called_once_with(settings.MAILCHIMP_LEARNERS_LIST_ID, {"members": users_data})
+        mock_func.assert_called_once_with(settings.MAILCHIMP_LEARNERS_LIST_ID,
+                                          {'update_existing': True, "members": users_data})
```

Comment:
I have updated the code to use single qoutes. Regarding storing dictionary in a variable, I think it would become a somewhat complicated problem to choose the appropriate name for that variable. Even in the actual command, the dictionary is passed like the way I passed in the test. Hope with the new formatting of this line the code is more readable. If not, it would be helpful if you can suggest a name for the dict variable.